db/migrations/000-create-games.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS Platform;
+DROP TABLE IF EXISTS Region;
+DROP TABLE IF EXISTS Version;
+
+DROP TABLE IF EXISTS RunnerRuns;
+DROP TABLE IF EXISTS Runs;
+DROP TABLE IF EXISTS Runner;
+
+DROP TABLE IF EXISTS Games;
+DROP TABLE IF EXISTS Category;
```

Comment:
generally you want your DB migrations files so they can be ran on-startup and they should just catch the schema up, or be no-ops (its why all CREATE TABLES only run if needed below).

With Cloudflare D1 it's a little bit difference since we won't be doing it on every worker invocation -- but I'd still probably recommend avoid doing this, if for no other reason than to avoid an accident.  

If people want to refresh their DB -- all they have to do is delete the sqlite file.